CONTRIBUTING.md

```diff
@@ -18,23 +20,176 @@ This project just started! Please wait for the first few versions to be released
 
 ## Responsibilities
 
-- Ensure cross-platform compatibility for every change that's accepted. Windows, Linux, Android, Xbox, iOS.
+- Ensure cross-platform compatibility for every change that's accepted. Windows, Linux, Android, iOS, Xbox, Playstation.
 - Ensure that code that goes into master branch meets all requirements from the requirements list below
 - Create issues for any major changes and enhancements that you wish to make. Discuss things transparently and get community feedback.
-- Don't add any classes to the codebase unless needed. Err on the side of using functions.
+- Don't add any classes to the codebase unless needed.
```

Comment:
I did not understand (and still dont) what it means. Thought it would be an error / unfinished sentence, but it seems to be a phrase? Anyhow, it seems to be separate from the first sentence of the bullet point